manual/README.md

```diff
@@ -42,6 +42,10 @@ things like deploying code or debugging complex parts of the stack.
 * [Find S3 upload folder](debugging/find-s3-upload-folder.md)
 * [Security log queries (Sharepoint)](https://nationalarchivesuk.sharepoint.com/:w:/r/sites/DA_Proj/Transfer%20Digital%20Records/Documentation/Developer%20manual/Security%20log%20queries.docx?d=w757b4033c718479eb0b1050a8328eb46&csf=1&web=1&e=fh1pOE)
 
+### Deploy the service unavailable page
+
+* [Deploy the service unavailable page](deploy-the-service-unavailable-page.md)
```

Comment:
Maybe put this under the `Admin Tasks` heading?

Reply:
That makes sense. I've moved the link.